llvm/lib/Target/AMDGPU/Disassembler/AMDGPUDisassembler.h

```diff
@@ -128,44 +128,6 @@ class AMDGPUDisassembler : public MCDisassembler {
 
   MCOperand errOperand(unsigned V, const Twine& ErrMsg) const;
 
-  template <typename InsnType>
-  DecodeStatus tryDecodeInst(const uint8_t *Table, MCInst &MI, InsnType Inst,
```

Comment:
Can you move the body to cpp and keep the declaration inside the class? So that you don't have to pass `this` to every call.

Reply:
Done.